apph-back/src/main/java/com/viseo/apph/domain/BaseEntity.java

```diff
@@ -0,0 +1,59 @@
+package com.viseo.apph.domain;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@MappedSuperclass
+public class BaseEntity implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    long id;
+    @Version
+    long version;
+
+    public BaseEntity() {
+        super();
+    }
+
+    public long getId() {
+        return id;
+    }
+
+    public BaseEntity setId(long id) {
+        this.id = id;
+        return this;
+    }
+
+    public long getVersion() {
+        return version;
+    }
+
+    public BaseEntity setVersion(long version) {
+        this.version = version;
+        return this;
+    }
+
+    @Override
+    public int hashCode() {
+        final int prime = 31;
+        int result = 1;
+        result = prime * result + (int) (id ^ (id >>> 32));
+        return result;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj)
+            return true;
+        if (obj == null)
+            return false;
+        if (getClass() != obj.getClass())
+            return false;
+        BaseEntity other = (BaseEntity) obj;
+        if (id != other.id)
+            return false;
+        return true;
```

Comment:
```suggestion
        return (id == other.id);
```
Dans les derniers commits, BaseEntity a été corrigé sur ces lignes. Il faut que tu merges les nouveaux commits de la branche dev.